docs/content/latest/explore/security/security.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Security
+headerTitle: Security
+linkTitle: Security
+description: Overview of security in YugabyteDB.
+headcontent: Security in YugabyteDB.
+image: /images/section_icons/index/secure.png
+aliases:
+  - /latest/explore/security/
+menu:
+  latest:
+    identifier: explore-security
+    parent: explore
+    weight: 930
+isTocNested: true
+showAsideToc: true
+---
+
+Like PostgreSQL, YugabyteDB provides security in multiple ways:
+
+* **Authentication** - limit access to the database to clients with proper credentials
+
+* **Authorization** - establish roles and privileges to restrict activities that clients can perform
```

Comment:
I'm not sure if it's "client" as opposed to "server", or "client" as a "user"

Reply:
Fixed